components/picker/index.css

```diff
@@ -135,7 +135,7 @@ governing permissions and limitations under the License.
   --spectrum-picker-spacing-bottom-to-text: var(--spectrum-component-bottom-to-text-300);
   --spectrum-picker-spacing-edge-to-text: var(--spectrum-component-edge-to-text-300);
   --spectrum-picker-spacing-text-to-icon: var(--spectrum-text-to-visual-300);
-  --spectrum-picker-spacing-text-to-alert-icon-inline-start: var(--spectrum-field-text-to-alert-icon-extra-large);
+  --spectrum-picker-spacing-text-to-alert-icon-inline-end: var(--spectrum-field-text-to-alert-icon-extra-large);
```

Comment:
I am not sure if I should actually do this change 😅 , made sense since the property will be `margin-inline-end`, but is it OK to change mod tokens like that?

Reply:
@mdt2 Generally you're right but I don't see any --mods in use in this case so I don't think we need to do any steps for deprecation. We don't recommend users overwrite --spectrum prefixed variables anyway since those are expected to change.